src/components/sidebar/index.tsx

```diff
@@ -32,6 +32,9 @@ import SidebarNavList from './components/sidebar-nav-list'
 import SidebarBackToLink from './components/sidebar-back-to-link'
 import SidebarMobileControls from './components/sidebar-mobile-controls'
 import s from './sidebar.module.css'
+import { OpenApiSidebarContents } from 'views/open-api-docs-view/components'
```

Comment:
Not a blocker for this PR, but if we're reusing this component we should provbably hoist it up to a shared component to make its usage clear.